src/router/httpd/visuals/sysctl.c

```diff
@@ -47,6 +47,7 @@ static void showsysctl(char *path, char *nvname, char *name, char *sysval, void
 	if (!path) {
 		if (p->cnt)
 			websWrite(wp, "</fieldset>\n");
+			websWrite(wp, "<br />\n");
```

Comment:
So Im hoping this is the result with this fix.

![Capture](https://user-images.githubusercontent.com/31389848/135708417-6770f0e9-11a0-4c87-aed3-3f3b429ac183.PNG)

Currently there are no breaks in between these fieldsets and dont really look consistent with other pages.

This will hopefully fix that. let me know if its the wrong place to add this break.